specification/columns/pricingquantity.md

```diff
@@ -2,7 +2,7 @@
 
 The Pricing Quantity represents the volume of a given SKU associated with a [*resource*](#glossary:resource) or [*service*](#glossary:service) used or purchased, based on the [Pricing Unit](#pricingunit). Distinct from [Usage Quantity](#usagequantity) (complementary to [Usage Unit](#usageunit)), it focuses on pricing and cost, not *resource* and *service* consumption.
 
-The PricingQuantity column MUST be present in the billing data. This column MUST be of type Decimal and MUST conform to [Numeric Format](#numericformat) requirements. The value MAY be negative in cases where [ChargeSubcategory](#chargesubcategory) is "Refund". This column MUST NOT be null if [ChargeCategory](#chargecategory) is "Usage" or "Purchase" and MAY be null for other ChargeCategory values. When unit prices are not null, multiplying PricingQuantity by a unit price MUST produce a result equal to the corresponding cost metric.
+The PricingQuantity column MUST be present in the billing data. This column MUST be of type Decimal and MUST conform to [Numeric Format](#numericformat) requirements. The value MAY be negative in cases where [ChargeClass](#chargeclass) is "Correction". This column MUST NOT be null if [ChargeCategory](#chargecategory) is "Usage" or "Purchase" and MAY be null for other ChargeCategory values. When unit prices are not null, multiplying PricingQuantity by a unit price MUST produce a result equal to the corresponding cost metric.
```

Comment:
```suggestion
The PricingQuantity column MUST be present in the billing data. This column MUST be of type Decimal and MUST conform to [Numeric Format](#numericformat) requirements. The value MAY be negative in cases where [ChargeClass](#chargeclass) is "Correction". This column MUST NOT be null if [ChargeCategory](#chargecategory) is "Usage" or "Purchase" and [ChargeClass](#chargeclass) is "Regular". This column MAY be null for other ChargeCategory values. When unit prices are not null, multiplying PricingQuantity by a unit price MUST produce a result equal to the corresponding cost metric.
```

Do we need to raise a separate issue to review all instances of `PricingCategory MUST NOT be null if [ChargeCategory] is "Usage" or "Purchase"` impacted by bulk correction concept?

Reply:
We discussed this at the TF-1 meeting on 4/16/2024 (see https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/issues/366). 
Agreed to create a separate PR.